### DISCUSSION TITLE
Frage zur Installation

### DISCUSSION THREAD

**Discussion by VolkerK62** [StefaE/PVForecast#18] [Q&A] [answered]
Hallo,
mangels ausreichender Englisch-Kenntnisse versuche ich es mal auf Deutsch.

Ich suche eine Möglichkeit die Vorhersage aus Solcast in Grafana zu visualisieren.
Dabei bin ich auf dieses Projekt gestoßen.
Habe ich das richtig verstanden, dass ich aus der requirements.txt nur den ersten Teil bis incl. Influx benötige?

Viele Grüße
Volker

**Comment by StefaE** [comment, marked as answer]
Hallo Volker,

… das hast Du richtig verstanden. Den Influx Server musst Du natuerlich separat installieren, wenn Du noch keinen hast.
(Ich denke, Google oder Bing sind mittlerweile perfekt, die englischen Texte zu übersetzen ;-) )

Viel Erfolg 
Stefan

**Reply by VolkerK62** [reply]
Tolle Sache. Hat geklappt.
Erst kam beim Scipt-Aufruf eine Fehlermeldung bzgl NumPy, aber der Hinweis hat mich zur Lösung geführt (libatlas-base-dev installiert)
Vielen Dank für dieses tolle Projekt.
Ich habe Wochen gesucht um Solcast-Daten nach Influx zu bekommen.

ps: der Tipp mit Google war gut. Chrome macht das Übersetzen sogar "automatisch"

**Comment by thunder312** [comment]
Wenn deutsch geht... :-)
Hallo STefanE,
erst mal danke für all deine Mühe.
Ich vermute ich bin am Gleichen Projekt dran wie Volker.
Ich habe nur das Problem, dass der Login bei SolCast nicht klappt.
Registriert habe ich mich und habe sowohl eine 16-stellige Resource ID (ja, 16 nicht 12; check mal deine Doku :-) ) 
als auch einen API key.
Beides habe ich in die config.ini eingetragen (Screenshot).
Wenn ich "python3 PVForecasts.py" ausführe, bekomme ich immer den Fehler:

_--v2.11.04--------------- Start (config.ini at 2024-02-11, 17:17:23 - local)
processing SolCast
Error - Method SolCast: 401: {"code":"unauthorized","message":"Unauthorized"}
------------------------- End (2024-02-11, 17:17:23 - local)_

Habe ich was vergessen einzutragen?

Kollegiale Grüsse,
Daniel
![solcast](https://github.com/StefaE/PVForecast/assets/26647148/86bbfa97-91de-444f-958e-b6946956bfdb)

**Reply by StefaE** [reply]
ok - 12 sollte 16 sein :-)

Aber sonst hast Du m.E. alles richtig gemacht (so dann die resource_id und api_key stimmen; mindestens api_key scheint case-sensitiv zu sein). Falls es nicht geht, musst Du Dich mit Solcast Support auseinander setzen.

**Reply by thunder312** [reply]
![image](https://github.com/StefaE/PVForecast/assets/26647148/a0863d3d-1634-4f16-aa4b-dc8aa35a4da8)
Den Teil in deiner Doku meinte ich. Zumindest bei mir hat die resource_id 16 Stellen.
Ich schau mal, ob ich jemanden vom support da erreiche.
Die Strings habe ich c&p kopiert, sollten also richtig sein.
Danke trotzdem.
